backend/drm/backend.c

```diff
@@ -57,16 +57,69 @@ bool wlr_backend_is_drm(struct wlr_backend *b) {
 	return b->impl == &backend_impl;
 }
 
+static void wlr_drm_connector_update_crtc_binding(struct wlr_drm_connector *conn,
+		struct wlr_drm_backend *drm) {
+	drmModeConnectorPtr drm_conn = drmModeGetConnector(drm->fd, conn->id);
```

Comment:
`drmModeConnector *drm_conn`
I really don't like typedefs on pointers.

Reply:
typedefs are not allowed at all in our codebase.

Reply:
It's not our typedef, but they provide both a struct and a pointer to struct typedef in libdrm for some stupid reason.